tools/javascript/lib/commands/test.ts

```diff
@@ -57,7 +58,27 @@ export function getTestCommand(
     default: 0.1,
     description:
       "The probability we use a random type regardless of the inferred type.",
-    group: commandGroup,
+    group: typeInferenceGroup,
+    hidden: false,
+    type: "number",
+  });
+
+  options.set("reuse-statement-probability", {
+    alias: [],
+    default: 0.8,
```

Comment:
```suggestion
    default: 0.9,
```

Related work uses 0.9 for this value